eng/Version.Details.xml

```diff
@@ -69,9 +70,15 @@
       <Uri>https://github.com/dotnet/fsharp</Uri>
       <Sha>13a21d4b08baa0e6b831cc2a7af872aeef8c22a5</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.fsharp" Version="5.0.2-beta.21321.4">
```

Comment:
The `Microsoft.FSharp.Compiler` cannot be adorned as source-build as it's version does not match the SB intemerdiate package version therefore the SB Intermediate dependency must be explicitly added.  Same applies to the roslyn-analyzers intermediate dependency that was added.